src/static/visualization/temperature-variations.js

```diff
@@ -59,26 +59,26 @@ function plot_temp_mean_variance(id, data) {
         text: 'mean'
     }
 
-    var trace_variance_min = {
+    var trace_st_dev_min = {
         x: months,
-        y: Array(months.length).fill(mean - variance / 2),
+        y: Array(months.length).fill(mean - st_dev / 2),
         type: 'scatter',
         mode: 'lines',
         fill: 'none',
         line: { color: 'rgba(31, 119, 180, 0.5)' },
         showlegend: false,
-        text: 'mean - variance / 2'
+        text: 'mean - st_dev / 2'
     }
 
-    var trace_variance_max = {
+    var trace_st_dev_max = {
         x: months,
-        y: Array(months.length).fill(mean + variance / 2),
-        name: 'variance',
+        y: Array(months.length).fill(mean + st_dev / 2),
+        name: 'Standard Deviation',
```

Comment:
Just making it lowercase so it matches the other names in the legend.

```suggestion
        name: 'standard deviation',
```